### PR TITLE
33 synchronous konserve io

### PR DESCRIPTION
**Pull request by whilo**
Use new konserve synchronous interface and remove blocking `core.async` operators.

### CODE REVIEW

File: test/hitchhiker/konserve_test.cljc [hunk 1]
```diff
@@ -151,4 +150,4 @@
                                       [del-freq (gen/tuple (gen/return :del)
                                                            (gen/no-shrink gen/int))]])
                                     num-ops)]
-                   (ha/<?? (ops-test ops universe-size)))))
\ No newline at end of file
+                   (ha/<?? (ops-test ops universe-size)))))
```
Comment (grischoun) [inline test/hitchhiker/konserve_test.cljc:153]: I can't see who calls this function. So is this ever run?

**Review by grischoun — COMMENTED** [review]
It looks ok. The test I could run works.
Though honestly I can't say much about the full PR. There are still many things I do not understand. 
E.g.,  what does a `KonserveAdr` represent?
I also wonder what `hash-node?` means?

**Comment by whilo** [comment]
`KonserveAddress` is a pointer to the underlying storage that allows the data (`IndexNode` or `DataNode`) to be loaded when it is needed while walking an iterator. `hash-node?` is a new configuration parameter that I have introduced to decide whether `hasch.core/uuid` is used to create a cryptographic hash on the value or whether to just use a random uuid (which is faster, but does not guarantee integrity [which does not matter yet, but will be crucial once snapshots are distributed and signed]).

File: test/hitchhiker/konserve_test.cljc (hunk 1 above)
Comment (whilo) [reply test/hitchhiker/konserve_test.cljc:153]: Good catch. It is still tested in `ClojureScript` with the recorded ops, but not here. I am not sure, but i think there was some problem with this test, but we should definitely run it again. @kordano Can you try whether you can call this function and create a test similar to the cljs one?

**Comment by kordano** [comment]
superseded by #35 since we moved to the single release branch approach.

**Pull request closed without merging**